sig-docs/README.md

```diff
@@ -48,6 +48,7 @@ subprojects, and resolve cross-subproject technical issues and decisions.
 
 * Tim Bannister (**[@sftim](https://github.com/sftim)**), The Scale Factory
 * Qiming Teng (**[@tengqm](https://github.com/tengqm)**), Sangfor Technologies
+* Kat Cosgrove (**[@katcosgrove](https://github.com/katcosgrove)**), Dell Technologies
```

Comment:
Sounds like we are not supposed to manually edit this file. It will be updated by some mysterious scripts.